resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -746,6 +746,25 @@ private[spark] object Config extends Logging {
       .checkValue(value => value > 0, "Gracefully shutdown period must be a positive time value")
       .createWithDefaultString("20s")
 
+  val KUBERNETES_MAX_EXECUTOR_FAILURES =
+    ConfigBuilder("spark.kubernetes.executor.maxNumFailures")
+      .doc("Spark exit if the number of failed executors exceeds this threshold. This " +
+        s"configuration only take effect when ${KUBERNETES_ALLOCATION_PODS_ALLOCATOR.key} " +
+        s"is set to 'direct'.")
+      .version("3.4.0")
+      .intConf
+      .createOptional
+
+  val KUBERNETES_EXECUTOR_ATTEMPT_FAILURE_VALIDITY_INTERVAL_MS =
```

Comment:
The val name is a little long ...

Reply:
there were such long name variables in existing configurations